File: BBIS.Application/DTOs/DonorRegistration/VerifyDonorResultDto.cs
namespace BBIS.Application.DTOs.DonorRegistration
{
    public class VerifyDonorResultDto
    {
        public Guid? donorRegistrationId { get; set; }
        public bool IsValid { get; set; }
        public string DeferralStatus { get; set; }
        public string Remarks { get; set; }
        public DonorDto Donor { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/DonorRegistration/VerifyDonorResultDto.ts
import { DonorDto } from "./DonorDto";

export interface IVerifyDonorResultDto {
    IsValid: boolean,
    DeferralStatus: string,
    Remarks: string | null,
    Donor: DonorDto,
    donorRegistrationId: Guid

}
  
export class VerifyDonorResultDto implements IVerifyDonorResultDto {
    IsValid: boolean = false;
    DeferralStatus: string = "";
    Remarks: string | null = null;
    Donor: DonorDto = new DonorDto();
    donorRegistrationId: Guid = "";
}
  